ale_linters/sh/shellcheck.vim

```diff
@@ -16,9 +16,21 @@ else
     let s:exclude_option = ''
 endif
 
+function! s:GetDialectArgument()
+    if exists('b:is_bash') && b:is_bash
+        return '-s bash'
+    elseif exists('b:is_sh') && b:is_sh
+        return '-s sh'
+    elseif exists('b:is_kornshell') && b:is_kornshell
+        return '-s ksh'
+    endif
```

Comment:
Ah interesting. Well there isn't an internal var like this for zsh since it's expected to be a different filetype. I'll check the ft first instead